src/app/app.component.ts

```diff
@@ -1,18 +1,50 @@
-import { Component } from '@angular/core';
+import {Component, OnInit} from '@angular/core';
 import { ConfigService } from './config.service';
+import {NavigationEnd, Router} from '@angular/router';
+import { environment } from '../environments/environment';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
 
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'JUI Web App';
 
   config;
 
-  constructor(private configService: ConfigService) {
+  constructor(private configService: ConfigService, private router: Router) {
```

Comment:
I would probally put this into a component called <app-google-analytic> and put in the shared module

Reply:
This is not working if I try in any other way, we would have further tickets in future to generate custom events for tracking. Will see to refactor that time.